data/src/main/java/com/microsoft/azure/kusto/data/auth/HttpClientWrapper.java

```diff
@@ -21,7 +21,7 @@ public HttpClientWrapper(HttpClient httpClient) {
 
     // Implementation of the synchronous HttpClient
     @Override
-    public IHttpResponse send(com.microsoft.aad.msal4j.HttpRequest httpRequest) {
+    public IHttpResponse send(com.microsoft.aad.msal4j.HttpRequest httpRequest) { // TODO: does this need to be changed?
```

Comment:
I think sendsync is ok here, since it's only for when it's through auth code, and even then only though stuff we still have to do through msal (Certificate stuff). Hopefully they update azure-identity with these features and we'll get rid of this and msal entirely.